### PR TITLE
update master

### PR DESCRIPTION
**Pull request by JeffGreiner-eaton**
Fixes BLUI-4594.

#### Changes proposed in this Pull Request:

- update master branch with blui-pr-actions plus the other changes
-
-

#### Screenshots / Screen Recording (if applicable)

-

#### To Test:

- once this is merged we can open a PR to master to verify labels and comment
 

#### Any specific feedback you are looking for?

-

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,62 +1,15 @@
...
+This package provides theming support for Eaton applications using the Brightlayer UI design system. It includes resources for developers using React w/ [Material UI](https://www.npmjs.com/package/@mui/material) version 5+ (prior versions of this package will work with Material UI version 4 - check the [Changelog](https://github.com/etn-ccis/blui-themes/blob/master/CHANGELOG.md) for details). This package comes with two theme options: a Blue theme (standard) and a Dark theme.
 
 For other frameworks, check out our related packages:
 
 -   [@brightlayer-ui/angular-themes](https://www.npmjs.com/package/@brightlayer-ui/angular-themes)
 -   [@brightlayer-ui/react-native-themes](https://www.npmjs.com/package/@brightlayer-ui/react-native-themes)
 
-## Installation
```
Comment (joebochill) [inline README.md:12]: I'm not a huge fan of completely gutting our instal/usage sections here. This is where a lot of developers are familiar with coming to find this information.

File: .circleci/config.yml [hunk 2]
```diff
@@ -36,19 +36,6 @@ jobs:
                     command:  |
                         yarn publish:package -b $CIRCLE_BRANCH
 
-    tag:
```
Comment (joebochill) [inline .circleci/config.yml:39]: Why are we removing the tag job?

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,33 +1,41 @@
 # Changelog
 
+## v7.2.0 (not released)
```
Comment (joebochill) [inline CHANGELOG.md:3]: Date will need to be updated

Comment (joebochill) [reply CHANGELOG.md:3]: Version will also need to be updated in package.json to actually publish it (master branch represents the latest published code...so if it's in master it MUST be published)

File: README.md (hunk 1 above)
Comment (JeffGreiner-eaton) [reply README.md:12]: ok will add back in a new PR. what about Updating From Version 5 ~> 6 section?

**Review by joebochill — APPROVED** [review]

**Review by daileytj — APPROVED** [review]

**Pull request merged**